aws-platform-ui-main/snapshots.tf

```diff
@@ -0,0 +1,53 @@
+locals {
+  snapshot_tags = {
+    # TODO: add per vol snapshot enabled tags
+    KubernetesCluster = module.eks_vpc.aws_eks_cluster_name
+  }
+}
+
+module "fabric_snapshots" {
+  source = "../ebs-dlm-snapshots"
+
+  count = var.enable_dlm_snapshots ? 1 : 0
+
+  description    = "${module.eks_vpc.aws_eks_cluster_name} every ${var.snapshot_frequency_hours} hour fabric snapshots"
+  target_tags    = local.snapshot_tags
+  role_arn       = aws_iam_role.dlm_snapshots[0].arn
+  luther_project = var.luther_project
+  luther_env     = var.luther_env
+
+  retain_count   = (24 / var.snapshot_frequency_hours) * var.snapshot_retention_days
```

Comment:
What happens if this isn't an integer? Do we need to add floor() or anything like that?